cmd/cli/plugin/airgapped-network/interface.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package util defines functions to pull/push/List images
+package main
+
+// util defines methods to pull/push/List images (similar to imgpkg)
+
+type ImgpkgUtil interface {
```

Comment:
It's more apt to interface a client rather than interfacing a util. `ImgPkgClient` would be a better candidate for an interface.